app/models/endpoint.rb

```diff
@@ -0,0 +1,25 @@
+class Endpoint < ActiveRecord::Base
+  belongs_to :resource, :polymorphic => true
+
+  default_value_for :verify_ssl, OpenSSL::SSL::VERIFY_PEER
+  validates :verify_ssl, :inclusion => {:in => [OpenSSL::SSL::VERIFY_NONE, OpenSSL::SSL::VERIFY_PEER]}
+
+  def verify_ssl=(val)
+    val = resolve_verify_ssl_value(val)
+    super
+  end
+
+  def verify_ssl?
+    verify_ssl != OpenSSL::SSL::VERIFY_NONE
+  end
```

Comment:
I'm pretty sure that was added for `foreman` - that may have been removed since.
Also, now that we default to `VERIFY_PEER`, it may no longer be needed